README.md

```diff
@@ -83,6 +83,35 @@ console.log('LDAP search filter: ', predicate.toLDAPFilterString());
 /* Prints "(|(hostname=spike)(latency>300))" */
 ```
 
+Please note however that trivial predicates cannot be serialized to LDAP search
```

Comment:
Does it make sense to explain that this is an LDAP limitation?  Something like: "Without knowing the LDAP object schema, it is not possible to generate a filter that matches all objects.  As a result, trivial predicates cannot be serialized as LDAP search filters."